pkgs/development/python-modules/qrcodegen/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  qrcodegen,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "qrcodegen";
+  pyproject = true;
+
+  inherit (qrcodegen)
+    version
+    src
+    meta
```

Comment:
inherit meta attributes separately

Reply:
I'm curious, is this suggestion a style choice or are there other considerations?

Reply:
Ah okay. Thanks.